### PR TITLE
Feature/add notificationservice

### PR DESCRIPTION
**Pull request by asyagraf**
-Add solution with name "LT.DigitalOffice.NotificationService";
-add directory with name "Solution items" and put the following files there: ".dockerignore", ".editorconfig", ".gitignore", "Common.Configurations.targets", "Dockerfile", "main.yml"
-add directory with name "src" and put the following items:
application wit name "NotificationService" (add appsettings.json, set Program.cs, Startup.cs), add following DLLs: NotificationService.Broker (add "Consumers" directory), NotificationService.Models.Dto

### CODE REVIEW

File: src/NotificationService.Models.Dto/NotificationService.Models.Dto.csproj [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="LT.DigitalOffice.Kernel" Version="1.1.53" />
+    <PackageReference Include="LT.DigitalOffice.Kernel.BrokerSupport" Version="1.0.1" />
+    <PackageReference Include="LT.DigitalOffice.Models" Version="1.0.56" />
+  </ItemGroup>
```
Comment (grinakate) [inline src/NotificationService.Models.Dto/NotificationService.Models.Dto.csproj]: maybe remove this and add a connection to Common.Configurations.targets?
same with other .csproj files

File: src/NotificationService/Program.cs [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+      }
+      finally
+      {
+        Log.CloseAndFlush();
+      }
+    }
+
+    public static IHostBuilder CreateHostBuilder(string[] args) =>
+       Host.CreateDefaultBuilder(args)
+           .UseSerilog()
```
Comment (MariyaSin) [inline src/NotificationService/Program.cs]: fix tab spase

File: src/NotificationService/Startup.cs [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+
+      services.AddBusinessObjects();
+
+      ConfigureMassTransit(services);
+    }
+
+    public void Configure(IApplicationBuilder app, ILoggerFactory loggerFactory)
+    {
+      app.UseForwardedHeaders();
+
```
Comment (MariyaSin) [inline src/NotificationService/Startup.cs:127]: remove empty line

File: src/NotificationService/appsettings.json [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    //RigthService
+    "CheckUserRightsEndpoint": "CRService_CheckUserRights",
+
+    //AuthService
+    "ValidateTokenEndpoint": "AuthService_ValidateToken",
+
+    //ImageService
+    "GetImagesEndpoint": "ImageService_GetImages"
+  },
+  "CheckTokenMiddleware": {
```
Comment (MariyaSin) [inline src/NotificationService/appsettings.json:51]: add "/apiinformation"

**Review by MariyaSin — APPROVED** [review]

**Review by grinakate — APPROVED** [review]

**Review by VladSpitsin — APPROVED** [review]

**Review by ndoriomedov — APPROVED** [review]

**Pull request merged**